util/flags.py

```diff
@@ -73,6 +77,7 @@ def create_flags():
     tf.app.flags.DEFINE_boolean ('export_tflite',    False,       'export a graph ready for TF Lite engine')
     tf.app.flags.DEFINE_boolean ('use_seq_length',   True,        'have sequence_length in the exported graph (will make tfcompile unhappy)')
     tf.app.flags.DEFINE_integer ('n_steps',          16,          'how many timesteps to process at once by the export graph, higher values mean more latency')
+    tf.app.flags.DEFINE_string  ('export_model_language', '',     'language the model was trained on. Gets embedded into exported model.')
```

Comment:
I'd like we get a different naming, it's confusing, I thought it was the language model we were embedding, I ad t ore-read carefully several time. Can the option be named "add_language_metadata" ?

Reply:
How about this:

`tf.app.flags.DEFINE_string  ('export_language',  '',          'language the model was trained on e.g. "en" or "English". Gets embedded into exported model.')`

Reply:
WFM, feels less confusing